tests/containers/podman_network_cni.pm

```diff
@@ -52,10 +52,36 @@ sub run() {
         assert_script_run('podman network inspect newnet1 --format "{{range .Subnets}}Subnet: {{.Subnet}} Gateway: {{.Gateway}}{{end}}"');
     }
 
+    #connect, disconnect & reload
+    unless ($old_podman) {
+        record_info('Prepare', 'Prepare three containers');
+        assert_script_run('podman run -id --rm --name container1 -p 1234:1234 registry.opensuse.org/opensuse/tumbleweed');
+        assert_script_run('podman run -id --rm --name container2 -p 1235:1235 registry.opensuse.org/opensuse/tumbleweed');
+        my $container_id = assert_script_run('podman run -id --rm --name container3 -p 8080:80 registry.opensuse.org/opensuse/nginx');
```

Comment:
Isn't `$container_id` empty?

Reply:
Yes, likely it should be `script_output`. See https://open.qa/api/testapi/#_assert_script_run and https://open.qa/api/testapi/#_script_output

Reply:
Fixed.